tests/testthat/tests-is_climatic_data.R

```diff
@@ -0,0 +1,43 @@
+library(testthat)
+library(R6)
+
+# Assume label used in metadata
+is_climatic_label <- "is_climatic"
+
+# Create a minimal mock object that includes is_climatic_data and mocks metadata methods
+MockData <- R6Class("MockData",
```

Comment:
Change this to 

```r
MockData <- R6::R6Class("MockData",
```

Then let's see if that works! (and remove above where you say `library(R6)`). 
I'm not sure why that is an issue, but looking at the warning it might be stemming from that. Your code looks good!